SOC/common/simulation.py

```diff
@@ -95,11 +95,20 @@ def plot_state(self, with_boundaries = False):
         Plots the current state of the simulation.
         """
         fig, ax = plt.subplots()
+
         if with_boundaries:
             values = self.values
         else:
             values = self.values[self.BOUNDARY_SIZE:-self.BOUNDARY_SIZE, self.BOUNDARY_SIZE:-self.BOUNDARY_SIZE]
+        
+        values_on_plot=False
+        if(values_on_plot):
+            for i  in range(values.shape[0]):
+                for j in range(values.shape[1]):
+                    ax.text(j,i,values[i,j], color="w", ha="center", va="center")     #trzeba dopracować format, jeśli miałoby być przydatne w prezentowaniu ułamkowych values
```

Comment:
Rozumiem że chcesz na każdym pikselku wypisać wartość? Podejrzewam że `seaborn` może już mieć coś takiego.

Swoją drogą, do sprawdzenia: kolejność pierwszych dwóch argumentów?

Reply:
> Rozumiem że chcesz na każdym pikselku wypisać wartość?

Właśnie się waham, czy jest sens to dobudowywać w tym miejscu. W niektórych przypadkach chce się widzieć konkretne wartości, ale wyświetlanie wartości na wykresie ma sens tylko przy małej ilości komórek, no i w sumie można utworzyć funkcję wyświetlającą lub zapisującą tablicę values niezależnie.
Na razie zrobiłem z `values_on_plot` argument opcjonalny.

Reply:
Generalnie... jak otwierasz matplotlibem wykres w okienku, lub w notebooku z interaktywnym interfejsem (`%matplotlib widget` bądź `%matplotlib notebook`), możesz zwykle najechać myszką na piksel i w lewym dolnym roku powinny się wyświetlić współrzedne i wartość w danym punkcie.

Do tego colorbar powinien wizualnie pokazywać, z jakiego rodzaju wartościami mamy do czynienia.

Reply:
No właśnie.
Usuwam.